machine-learning/app/config.py

```diff
@@ -27,6 +27,7 @@ class Settings(BaseSettings):
     model_inter_op_threads: int = 0
     model_intra_op_threads: int = 0
     ann: bool = True
+    preload: str = ""
```

Comment:
I now made the setting to be list[tuple[ModelType, str]] which parses string like: '[["clip", "ViT-B-32__openai"], ["facial-recognition", "buffalo_s"]]'. Do you think that's enough or should I implement a custom parser?

Reply:
That's great! I don't think we need to do any other parsing here

Reply:
Is there a way to have Pydantic accept a slightly less verbose format? I quite liked the initial `CLIP:ViT-B-32__openai,FACIAL_RECOGNITION:buffalo_s`, this new format seems more prone to typoes and such.

Reply:
Pydantic supports this too:

`PRELOAD_MODEL__CLIP=['ViT-B-32__openai']`
`PRELOAD_MODEL__FACIAL_RECOGNITION=['buffalo_l']`

Reply:
Is tere actually a need for multiple models preloaded for one type? If not then it could also just be something like:
```
PRELOAD__CLIP=ViT-B-32__openai
PRELOAD__FACIAL_RECOGNITION=buffalo_l
```